docs/user/commandline_tools.rst

```diff
@@ -0,0 +1,338 @@
+
+Overview
+--------
+
+GeoMesa Tools is a set of command line tools to add feature management
+functions, query planning and explanation, ingest, and export abilities from
+the command line.  
+
+Installation
+------------
+
+See :ref:`setting_up_commandline`.
+
+Running the command line tools
+------------------------------
+
+Run the ``geomesa`` without any arguments to produce the following usage text::
+
+    $ geomesa
+    Warning: GEOMESA_HOME is not set, using /opt/devel/src/geomesa/geomesa-dist/target/geomesa-1.2.0-SNAPSHOT/dist/tools/geomesa-tools-1.2.0-SNAPSHOT
+    Usage: geomesa [command] [command options]
+      Commands:
+        create              Create a feature definition in a GeoMesa catalog
+        deletecatalog       Delete a GeoMesa catalog completely (and all features in it)
+        deleteraster        Delete a GeoMesa Raster Table
+        describe            Describe the attributes of a given feature in GeoMesa
+        env                 Examine the current GeoMesa environment
+        explain             Explain how a GeoMesa query will be executed
+        export              Export a GeoMesa feature
+        getsft              Get the SimpleFeatureType of a feature
+        help                Show help
+        ingest              Ingest/convert various file formats into GeoMesa
+        ingestraster        Ingest a raster file or raster files in a directory into GeoMesa
+        list                List GeoMesa features for a given catalog
+        queryrasterstats    Export queries and statistics about the last X number of queries to a CSV file.
+        removeschema        Remove a schema and associated features from a GeoMesa catalog
+        tableconf           Perform table configuration operations
+        version             GeoMesa Version
+
+This usage text lists the available commands. To see help for an individual command,
+run ``geomesa help <command-name>``, which for example will give you something like this::
+
+    $ geomesa help list
+    Warning: GEOMESA_HOME is not set, using /opt/devel/src/geomesa/geomesa-dist/target/geomesa-1.2.0-SNAPSHOT/dist/tools/geomesa-tools-1.2.0-SNAPSHOT
+    List GeoMesa features for a given catalog
+    Usage: list [options]
+      Options:
+        --auths
+           Accumulo authorizations
+      * -c, --catalog
+           Catalog table name for GeoMesa
+        -i, --instance
+           Accumulo instance name
+        --mock
+           Run everything with a mock accumulo instance instead of a real one
+           Default: false
+        -p, --password
+           Accumulo password (will prompt if not supplied)
+      * -u, --user
+           Accumulo user name
+        --visibilities
+           Accumulo scan visibilities
+        -z, --zookeepers
+           Zookeepers (host[:port], comma separated)
+
+The Accumulo username and password is required for each command. Specify the
+username and password in each command by using `-u`/`--username` and `-p`/
```

Comment:
in the web page, this looks like -ul and -pl instead of -u <slash> and -p <slash> - maybe change the slash to an 'or'?